airbyte-webapp/src/components/ui/NextTable/NextTable.module.scss

```diff
@@ -0,0 +1,74 @@
+@use "scss/colors";
+@use "scss/variables" as vars;
+
+// ------- <table/> --------
+.table {
+  border-spacing: 0;
+  width: 100%;
+  max-width: 100%;
+  border-radius: vars.$border-radius-md;
+}
+
+.thead {
+  position: sticky;
+  top: 0;
+}
+
+// --------- <th/> ---------
+.th {
+  background: colors.$grey-50;
+  color: colors.$grey;
+  text-transform: uppercase;
+  border-bottom: none;
+  text-align: left;
+  font-size: 10px;
+  line-height: 12px;
+  font-weight: 600;
+
+  &:first-child {
+    padding-left: calc(vars.$spacing-2xl + vars.$spacing-sm);
+    border-radius: vars.$border-radius-md 0 0;
+  }
+
+  &:last-child {
+    padding-left: vars.$spacing-lg;
+    border-radius: 0 vars.$border-radius-md 0 0;
+  }
+}
+
+// --------- <tr/> ---------
+.tr {
+  background: colors.$white;
+  cursor: auto;
+
+  &:hover {
+    background-color: colors.$grey-50;
+  }
+}
+
+// --------- <td/> ---------
+
+.td {
+  padding: vars.$spacing-lg calc(vars.$spacing-md + vars.$spacing-xs);
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: normal;
+  color: colors.$dark-blue;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  border-bottom: vars.$border-thin solid colors.$grey-100;
```

Comment:
The border-bottom in figma is grey-50

Reply:
Fixed via extending the base NextTable component to support style override